packages/node/src/indexer/indexer.manager.ts

```diff
@@ -214,13 +226,17 @@ export class IndexerManager {
     });
     this.filteredDataSources = this.filterDataSources(startHeight);
     this.fetchService.register((block) => this.indexBlock(block));
+  }
 
-    if (this.nodeConfig.proofOfIndex) {
-      void this.mmrService.syncFileBaseFromPoi().catch((err) => {
+  private setBlockOffset(offset: number): void {
+    this.blockOffset = offset;
+    logger.info(`set blockoffset to ${offset}`);
+    void this.mmrService
```

Comment:
What's the reason for not awaiting this?

Reply:
I assume you are asking  the `mmr.syncFileBaseFromPoi` here, while sync the between poi and filebased mmr,  this should be independent and we try to not block indexing service.